apps/detect/src/components/hosts/host-list.tsx

```diff
@@ -60,6 +72,17 @@ const HostRow: React.FC<{
         <p>{endpoint.endpoint_id}</p>
 
         <div className={styles.details}>
+          {platform && (
+            <HostDetail
+              icon={<VariantIcon platform={platform.split("-")[0]} />}
+              comment={platform.split("-")[0]}
+            />
+          )}
+          {probe?.tags &&
+            probe.tags.map(
```

Comment:
I think this should do the same thing
```suggestion
          {probe?.tags?.map(
```